index.html

```diff
@@ -66,67 +66,139 @@
+            </div>
+            <div class="view-more">
+                <a href="">SEE THE WHOLE PROGRAM</a>
+            </div>
+          </div>
         </div>
 
-        <div>
-            <div title="title">
-                <h3>Main Program</h3>
-                <hr>
+        <div class="container">
+            <div class="title">
+                <h3>Featured Speakers</h3>
+                <hr class="hr">
             </div>
             <div>
-                <div class="row">
-                    <div class="col">
-                        <img src="" alt="">
-                        <div class="desc">
-                            <h4>Agasha Bornittah</h4>
-                            <p>Software Engineer</p>
-                            <p>
-                                Lorem Ipsum is simply dummy text of the printing and typesetting industry.
-                            </p>
+                <div class="coach-row">
+                    <div class="card">
+                        <div class="card-header">
+                            <img src="./images/bg-5.jpg" alt="user" class="bg-image">
+                            <img src="./images/user1.png" alt="user" class="user-image">
+                        </div>
+                        <div class="card-body">
+                            <div  class="card-title">
+                                <h4>Agasha Bornittah</h4>
+                                <p class="orange">Software Engineer</p>
+                                <hr>
+                            </div>
+                            <div class="card-description">
+                                <p>
+                                    Lorem Ipsum is simply dummy text of the printing and typesetting industry.
+                                </p>
+                            </div>
                         </div>
                     </div>
+                    <div class="card">
+                        <div class="card-header">
+                            <img src="./images/bg-5.jpg" alt="user" class="bg-image">
+                            <img src="./images/user1.png" alt="user" class="user-image">
+                        </div>
+                        <div class="card-body">
+                            <div  class="card-title">
+                                <h4>Agasha Bornittah</h4>
+                                <p class="orange">Software Engineer</p>
+                                <hr>
+                            </div>
+                            <div class="card-description">
+                                <p>
+                                    Lorem Ipsum is simply dummy text of the printing and typesetting industry.
+                                </p>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="more-btn">
+                    <a href="">More</a>
```

Comment:
- [ ] Good job adding the `more` button but in the mobile view only 2 speakers should be displayed together with the `more` button. Kindly address this issue.
- [ ] (OPTIONAL) To improve the user interface of your page, you can implement the functionality of the `more` button. In the mobile view, when a user clicks on the `more` button, the rest of the speakers should be displayed 💯 
- Your design:
![Annotation 2022-05-26 085358](https://user-images.githubusercontent.com/85113567/170448384-e3b43b92-eaaa-4669-84d6-8a60cf2c499e.jpg)
- Required design:
![Annotation 2022-05-26 085423](https://user-images.githubusercontent.com/85113567/170448428-98f027af-6399-45fb-8bd9-3f8bc1a89226.jpg)